downstream/modules/platform/proc-aap-controller-backup.adoc

```diff
@@ -0,0 +1,42 @@
+[id="aap-controller-backup"]
+
+= Backing up the {ControllerNameStart} deployment
+
+[role=_abstract]
+Use this procedure to back up a deployment of the controller, including jobs, inventories, and credentials.
+
+.Prerequisites
+
+* You must be authenticated with an Openshift cluster.
```

Comment:
Same question as previous section about OpenShift and whether is should be an attribute. If not, ignore the questions.

Reply:
> Same question as previous section about OpenShift and whether is should be an attribute. If not, ignore the questions.

See above: OCP refers to the OpenShift Container Platform but OpenShift alone, I'm referring to a cluster. I'll ask in the team call whether a generic attribute should be created for the vanilla OpenShift term.